src/Athena/Views/Home/Add_Classes.cshtml

```diff
@@ -0,0 +1,21 @@
+﻿@{
+    ViewData["Title"] = "Add Classes";
+}
+
+<p>Use this area to provide additional information.</p>
+
+<!--http://jordnkr.github.io/collapsible/-->
+<div id="first">
+    <h1 id="specific" class="btn" style="width:30em;background-color:navy">Degree Specific Courses</h1>
```

Comment:
Use the [grid system](http://materializecss.com/grid.html) for size and layout control instead of manually specifying position or size.